app/src/main/java/org/sil/gatherwords/EntryFragment.java

```diff
@@ -4,14 +4,17 @@
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
+import android.support.design.widget.FloatingActionButton;
 import android.support.v4.app.Fragment;
 import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
 import android.widget.ListView;
 import android.widget.TextView;
+import android.widget.Toast;
```

Comment:
I missed removing this import on my check through. May want to delete it in merge.

Reply:
Deleted.